travis/git-crypt-setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+
+if [ "$#" -ne 0 ]; then
+    echo "Add Travis CI to a git-crypt set up for a repository"
+	echo "Will create an uncommitted diff."
+    echo "Usage: $0"
+	exit 1
+fi
+
+email="travis-ci@example.com"
+
+if ! travis -v; then
+	echo "Please install the \`travis\` cli"
+	exit 2
+fi
+
+if ! gpg --version; then
+	echo "Please install \`gpg\`"
+	exit 3
+fi
+
+if ! git-crypt version; then
+	echo "Please install the \`git-crypt\` binary"
+	exit 4
+fi
+
+GNUPGHOME="$(mktemp -d)"
+export GNUPGHOME
+cat >/tmp/key-generation.txt <<EOF
+     %echo Generating a basic OpenPGP key
+     Key-Type: RSA
+     Key-Length: 2048
+     Subkey-Length: 2048
+     Name-Real: Travis CI
+     Name-Email: ${email}
+     Expire-Date: 0
+     %commit
+     %echo done
+EOF
+gpg --batch --gen-key /tmp/key-generation.txt
+
+#gpg --armor --export "${email}" > "travis-ci.pub"
```

Comment:
Do we need this line?

Reply:
It's to dump the public key during debug, but we don't really need it because the only thing to be stored in the repository is a private key for Travis CI to use. So I guess we can just remove it.